docs/dev/run_the_etl.rst

```diff
@@ -11,24 +11,182 @@ just want to use already processed data.
+are collections of assets, resources, IO managers and jobs that can
+be loaded into dagit and executed.
+
+There are two main Definitions in the PUDL processing pipeline:
+
+1. ``pudl.ferc_to_sqlite`` :doc:`converts the FERC Form 1 DBF/XBRL files <clone_ferc1>`
+   into a single large `SQLite <https://sqlite.org>`__ database so that the data is
+   easier to extract, and so all of the raw FERC Form 1 data is available in a modern
+   format. You must run this script before you can run ``pudl_etl``.
+2. ``pudl.etl`` coordinates the "Extract, Transform, Load" process that processes
    20+ years worth of data from the FERC Form 1 database, dozens of EIA spreadsheets,
    and the thousands of CSV files that make up the EPA CEMS hourly emissions data into
    a clean, well normalized SQLite database (for the FERC and EIA data), and an `Apache
    Parquet <https://parquet.apache.org/>`__ dataset that is partitioned by state and
    year (for the EPA CEMS).
 
+Both definitions have two preconfigured jobs:
+  - ``etl_fast`` processes one year of data
+  - ``etl_full`` processes all years of data
+
+.. _run-dagit:
+
+Running the ETL with Dagit
+--------------------------
+Dagster needs a directory to store run logs and some assets. Create a new directory
+outside of the pudl respository directory called ``dagster_home/``. Then set the
+``DAGSTER_HOME`` variable to the path of the new directory:
+
+.. code-block:: console
+
+    mamba env config vars set DAGSTER_HOME=</path/to/dagster_home/directory/>
+    mamba activate pudl-dev
+
+Once ``DAGSTER_HOME`` is set, launch Dagit by running:
+
+.. code-block::
+
+    make dagit
+
+This will launch Dagit at http://localhost:3000/. You should see
+a window that looks like this:
+
+.. image:: ../images/dagit_home.png
+  :width: 800
+  :alt: Dagit home
+
+Click the hamburger button in the upper left to view the definitions,
+assets and jobs. First, you'll need to create the raw FERC databases by
+clicking on one of the ``pudl.ferc_to_sqlite`` jobs. Then select "Launchpad"
+where you can adjust the years to extract for each dataset. Then click
+"Launch Run" in the lower right hand corner of the window. Dagit will
+take you to a new window that provides information about the status of
+the job. The bottom part of the window contains dagster logs. You can
+view logs from the ``pudl`` package in the CLI window the dagit process
+is running in.
+
+Once the raw FERC databases are created by a ``pudl.ferc_to_sqlite`` job,
+you can execute the main PUDL ETL.
+
+.. note::
+
+  Make sure you've extracted the raw FERC years you are planning to process
+  with the main PUDL ETL. Jobs in the ``pudl.etl`` definition will fail if
+  the raw FERC database are missing requested years. For example, if you want
+  to process all years available in the ``pudl.etl`` definition make sure
+  you've extracted all years of the raw FERC data.
+
+Select one of the ``pudl.etl`` jobs.
```

Comment:
Can we run select parts of the ETL now? How does this work for inter-related datasets like EIA or CEMS?

Reply:
Yes we can run parts of the ETL. You won't be able to run an asset if an upstream asset hasn't been executed. For example, you can't run the hourly_emissions_epacems asset if the EIA-EPA crosswalk and plants_eia860 assets haven't been executed.